lib/key_generators/rsa_key_generator.dart

```diff
@@ -80,32 +80,32 @@ class RSAKeyGenerator implements KeyGenerator {
       for ( ; ; ) {
         q = generateProbablePrime(qbitlength, 1, _random);
 
-        if ((q - p).abs().bitLength() < mindiffbits) {
+        if ((q - p).abs().bitLength < mindiffbits) {
           continue;
         }
 
-        if (q.mod(e) == 1) {
```

Comment:
Changing `mod` for `modInverse`, that seems not correct.. I think the `BigInt` method has the `%` operator implemented. So I think you can do `q % e == BigInt.one`.